src/jade/resources/default_cfg/benchmarks_pp/excel/ALARM-CF-LAB.yaml

```
@@ -0,0 +1,83 @@
+    - Count rates 5 inches (without Cd)
+  comparison_type: percentage
+  table_type: pivot
+  x: Cells
+  y: Case
+  value: Value
+  add_error: True
+  change_col_names: {'Cells': 'Detector position', 'Case': 'Labyrinth case'}
+  conditional_formatting: {"red": 20, "orange": 10, "yellow": 5}
+#
+  8" Count Rates(%):
+  results:
```

YAMLlint (1.37.1)

[error] duplication of key "results" in mapping

(key-duplicates)